src/users.js

```diff
@@ -0,0 +1,147 @@
+const axios = require('axios')
+const cheerio = require('cheerio')
+
+const BASE_URI = 'https://myanimelist.net/profile/'
+
+/* the method that it's used in order to use to parse the page
+   and get all the info we want
+ */
+const parsePage = ($, name) => {
+  const pfp = $('#content .user-image img') // getting the profile picture page section
+  const status1 = $('#content .user-profile .user-status-title') // getting the status titles page section
+  const status2 = $('#content .user-profile .user-status-data') // getting the status data page section
+  const result = [] // we will put here all the properties of the final object
+  // pushing some basic properties and values
+  result.push({ Username: name })
+  result.push({ ProfilePictureLink: $(pfp).attr('data-src').trim() })
+  result.push({ LastOnline: $(status2[0]).text() })
+  // loop for the status
+  let i = 1
+  const arrayLength = status1.length
+  while (i < arrayLength - 1) {
+    const val = $(status1[i]).text()
+    switch (val) {
+      case 'Gender':
+        result.push({
+          Gender: $(status2[i]).text()
+        })
+        break
+      case 'Birthday':
+        result.push({
+          Birthday: $(status2[i]).text()
+        })
+        break
+      case 'Location':
+        result.push({
+          Location: $(status2[i]).text()
+        })
+        break
+      case 'Joined':
+        result.push({
+          Joined: $(status2[i]).text()
+        })
+    }
+    i++
+  }
```

Comment:
```suggestion
  const pfp = $('#content .user-image img') // getting the profile picture page section
  const statusTitles = $('#content .user-profile .user-status-title') // getting the status titles page section
  const statusData = $('#content .user-profile .user-status-data') // getting the status data page section
  const result = [] // we will put here all the properties of the final object
  // pushing some basic properties and values
  result.push({ Username: name })
  result.push({ ProfilePictureLink: $(pfp).attr('data-src').trim() })
  result.push({ LastOnline: $(statusData[0]).text() })

  statusTitles.each(function (index, status) {
    result.push({
      [$(status).text()]: $(statusData[index]).text()
    })
  })
```

Reply:
I don't think that's a good solution. Here are the results from my account: 
![Untitled](https://user-images.githubusercontent.com/31649672/204339637-07277d61-0f26-4d61-81e5-c736400ce9c3.png)

Reply:
Well then, just add a filter on `$(status).text()` to have only the ones you want

Reply:
Nevermind, just added it on d357f88